### ISSUE TITLE
Replace the Firefox word in all strings with a placeholder

### ISSUE THREAD

**Issue by Amejia481** [mozilla-mobile/fenix#22189]
We would like to replace the word "**Firefox**" in all our strings with a string placeholder (only on `/app/src/main/res/values/strings.xml`). For example:

**This is what we have:** 😞 

````xml
    <string name="synced_tabs_no_tabs">You don’t have any tabs open in Firefox on your other devices.</string>
````

**This is what we need:** 😄 
The string above, we want replace the word "**Firefox**" with a placeholder that we will replace in code:

````xml
    <string name="synced_tabs_no_tabs" moz:removedIn="95" tools:ignore="UnusedResources">You don’t have any tabs open in Firefox on your other devices.</string>
    <string name="synced_tabs_no_tabs_2">You don’t have any tabs open in %1$s on your other devices.</string>
````

Replace each reference of the string in code using the string placeholder `app_name`:

```kotlin

        val appName = resources.getString(R.string.app_name)
        description.text = getString(
            R.string.synced_tabs_no_tabs_2, appName
        )
```

From the code above, you can notice:
1. We don't actually update the string `synced_tabs_no_tabs` we created a new string adding a `_#` (a underscore with number, starting on 2), and added the properties `moz:removedIn="95" tools:ignore="UnusedResources"` to `synced_tabs_no_tabs`. We do this, for limitation described [here](https://github.com/mozilla-mobile/fenix/wiki/Remove-strings).
2. In code we replace the place the placeholder with the string `app_name`, we also have to make sure we replace all reference of the old string (`synced_tabs_no_tabs`) with the new string (`synced_tabs_no_tabs_2`) both in code and XML layouts.

For more context see https://github.com/mozilla-l10n/android-l10n/pull/393#discussion_r728314345

**Comment by grigoryk** [comment]
@Amejia481 I'm assuming this is what you meant, but just to clarify:
I think in case of intentionally hardcoded strings, e.g. something like `Firefox Account` or `Firefox Sync`, we should leave them as-is. That is, we don't have a `Firefox Preview Sync` service.

**Comment by Amejia481** [comment]
@grigoryk thanks for clarifying, yeah, you are right. @S10MC2015 as @grigoryk mentioned we want to keep the ones that are referring to specific Firefox services as they are. We only want to update strings that are related to the app's name.

**Comment by Alexandru2909** [comment]
Hi, I would like to work on this issue!

**Comment by Amejia481** [comment]
@Alexandru2909 thanks for interesting on this bug, lets just confirm with @S10MC2015. if we don't get any update after a day, please feel free to work on it, as it's been a long time since the last update.

**Comment by S10MC2015** [comment]
@Alexandru2909 can work on it. 

Sorry I haven't done any work. I don't think I'll have the time to work on for a while either.

**Comment by sflorean** [comment]
Tested this on Nightly 99 and here are the results:
**Case 1**
- signed with the same Firefox account on Nightly 2 devices with no open tabs: "You don't have any tabs open in Firefox Nightly on your other devices" is displayed.
 **Case 2**
- signed with the same Firefox account on Nightly and 2 different profiles of Firefox Release with no open tabs: "You don't have any tabs open in Firefox Nightly on your other devices" is displayed.

For case 2, the placeholder shouldn't be Firefox only? since the other tabs are on 2 different profiles of Firefox Release.

@Amejia481 @Alexandru2909

**Comment by Amejia481** [comment]
@sflorean 👋🏽 

Thanks for taking a look!
Yes, I think so as it looks we are referring to tabs in any Firefox,@Alexandru2909 could also help to clarify.

**Comment by Alexandru2909** [comment]
I agree, this seems to be the case. Could the same reasoning be applied to "Sync Firefox between devices"? or is it better to use `app_name` here?

**Comment by Amejia481** [comment]
Yes, I guess 😅 , it's a bit tricky but I think it was the intention of the message.

**Comment by SoftVision-LorandJanos** [comment]
Verified as fixed on the latest Nightly 100.0a1 (2022-04-01) build.
Device used: Oppo Reno 6 (Android 12).
